setup.md

```diff
@@ -67,9 +67,7 @@ https://docs.microsoft.com/en-us/powershell/scripting/learn/remoting/ssh-remotin
 
 ### macOS
 
-For a Mac computer running macOS Mojave or earlier releases, the default Unix Shell is Bash. For a
-Mac computer running macOS Catalina or later releases, the default Unix Shell is Zsh. Your default
-shell is available via the Terminal program within your Utilities folder.
+On macOS, the default Unix Shell is accessible by running the Terminal program from the `/Application/Utilities` folder in Finder.
```

Comment:
Maybe comment on the fact, that the terminal can be found through the search bar on macOS as well? (would be more inline with the description for Linux.)